.github/workflows/build-and-push.yml

```diff
@@ -8,54 +8,17 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        version: [14, 16]
+        dockerfile_path: [ "14", "16" ]
     steps:
-      - uses: actions/checkout@v2
-
-      - name: Get base image name
-        id: base-image-name
-        run: |
-          # This command returns row with BASE_IMAGE_NAME
-          row=$(grep "BASE_IMAGE_NAME" Makefile)
-          # Return only base image name
-          BASE_IMAGE_ROW=${row/BASE_IMAGE_NAME = /}
-          echo ::set-output name=image_name::$BASE_IMAGE_ROW
-
-      - name: Get short version
-        id: short_version
-        run: |
-          ver="${{ matrix.version }}"
-          echo "::set-output name=SHORT_VER::${ver//./}"
-
-      - name: Check if .exclude-centos7 is present in version directory
-        id: check_exclude_centos7_file
-        # https://github.com/marketplace/actions/file-existence
-        uses: andstor/file-existence-action@v1
-        with:
-          files: "${{ matrix.version }}/.exclude-centos7"
-
-      - name: Build CentOS7 image
-        if: steps.check_exclude_centos7_file.outputs.files_exists == 'false'
-        id: build-image
-        # https://github.com/marketplace/actions/buildah-build
-        uses: redhat-actions/buildah-build@v2
-        with:
-          dockerfiles: ${{ matrix.version }}/Dockerfile
-          image: ${{ steps.base-image-name.outputs.image_name}}-${{ steps.short_version.outputs.SHORT_VER }}-centos7
-          context: ${{ matrix.version }}
-          tags: latest 1 ${{ github.sha }}
-
-      - name: Push CentOS7 image to Quay.io
-        if: steps.check_exclude_centos7_file.outputs.files_exists == 'false'
-        id: push-to-quay
-        uses: redhat-actions/push-to-registry@v2.2
+      - name: Build and push to quay.io registry
+        uses: sclorg/build-and-push-action@v1
         with:
-          image: ${{ steps.build-image.outputs.image }}
-          tags: ${{ steps.build-image.outputs.tags }}
-          registry: quay.io/centos7
-          username: ${{ secrets.QUAY_IMAGE_BUILDER_USERNAME }}
-          password: ${{ secrets.QUAY_IMAGE_BUILDER_TOKEN }}
-
-      - name: Print image url
-        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
+          registry: "quay.io"
+          registry_namespace: "centos7"
```

Comment:
What about C9S? This repository belongs to sclorg.

Reply:
Thanks for the review.
I only rewrote the action as it was. I didn't want to add any new functionality. 

I would suggest creating other PR for adding c9s support, to test it separately, but if you really want, I can add it here, WDYT?